### DISCUSSION TITLE
Make Åå easier to type

### DISCUSSION THREAD

**Discussion by eljestliv** [qwerty-fr/qwerty-fr#74] [Ideas]
I speak Swedish, English, and French. I’ve been using qwerty-fr whenever I type French or English, but the difficulty of writing Å is preventing me from switching over permanently.

In my opinion, AltGr/Option + F should write the letter å.

**Comment by devnoname120** [comment]
I guess `F` would stand for `Finnish` which could more or less justify having `å` on that letter (it's part of the Finnish and Swedish alphabet). I'm not really fond of that location though because it really looks like out of place.

Another possibility would be to kick out the `æ` and put it elsewhere (where?). But then `œ` would have to be moved as well because both should follow the same logic…

What do you think?

---

I don't speak Finnish, but how difficult really is it to type `å`/`Å`?

With my right hand I can just press <kbd>Option ⌥</kbd> <kbd>Shift ⇧</kbd> <kbd>;</kbd>  then release <kbd>Option ⌥</kbd> and press <kbd>a</kbd> with my left hand in order to type `Å`. Is this more difficult than it looks like?

**Reply by CouldBeMathijs** [reply]
I like qwery-fr specifically to stop me from having to use dead keys, imo it would be handy to have the ability to type å and Å without deadkeys would be a plus, even if the only justification for putting it on F was to not screw up other peoples muscle memory.

**Comment by eljestliv** [comment]
I just picked F because it’s a key with no other uses that’s still comfortable. I wouldn’t mind if it was something else either, just as long as there’s a separate combination for it without the ring dead key. Like I appreciate there being a separate key for Ø even though it’s kinda hard to reach.

Finnish doesn’t use Å, by the way. Swedish uses it natively, Norwegian adopted it in 1917 and Danish in 1948 and it’s implemented in modern spellings but it’s sometimes written as Aa in names for people and places. Compare Danish Aalborg with Swedish Ålborg.

**Comment by Nibb31** [comment]
See my suggestion here:
https://github.com/qwerty-fr/qwerty-fr/discussions/66

The Å is used internationally as the standard abbreviation of angstrom:
https://en.wikipedia.org/wiki/Angstrom

**Comment by bgolinvaux** [comment]
Hello,
In my opinion, kicking « œ » would be very inconvenient for French users being able to type « cœur », « bœuf », « œuf » or « œil » easily, while these are commonly used words. 😊

Apologies if I got the proposal wrong, @devnoname120.

**Comment by Nibb31** [comment]
I would put œ on the o key, as in my fork here:
https://github.com/Nibb31/qwerty-fr

It has the following dead key:
å = Shift-AltGr-° + a
Å = Shift-AltGr-° + Shift-A
